content/1_docs/1_guide/8_users/3_permissions/guide.txt

```diff
@@ -52,14 +57,16 @@ permissions:
     changeRole: false
     delete: false
     update: false
-  files:
-    create: true
-    changeName: true
-    delete: true
-    replace: true
-    update: false
+  users: false
```

Comment:
The users options are missing if these are supposed to be all possible actions.

Reply:
✅